evap/student/templates/student_vote.html

```diff
@@ -202,6 +242,30 @@ <h3 class="mb-3">{{ course.name }} ({{ course.semester.name }})</h3>
                     presaveButton.text(originalPresaveText);
                 }, 2500);
             }
+
+            $("#droppedButton").on("click", function (event) {
+                $("#droppedCard").toggle("show");
+                $("#droppedButtonCard").toggle("show");
+
+                markUnansweredAsNoAnswerFor("general");
+
+                var contributorIds = [
+                {% for contributor, __ , __ , __ in contributor_form_groups %}
+                    {{ contributor.id }},
+                {% endfor %}
+                ];
+
+                for(var i = 0; i <  contributorIds.length; i++) {
+                    markNoAnswerFor(contributorIds[i])
+                }
```

Comment:
I understand that you're using building on top of an older function here, but passing in the IDs in the template generation to call the function for all ids feels wrong for me. What do you think about giving all contributor-cards that currently have a "#vote-area-X"-id a common class and writing a javascript method that iterates over all these divs by class? Imo that would be cleaner. I don't have much JS experience though. @karyon what do you think?